lib/fog/compute/google/models/network.rb

```diff
@@ -36,12 +36,10 @@ def destroy(async = true)
           operation
         end
 
-        # Returns a ready API structure for insert_instance, used in insert_server request.
-        def get_as_interface_config(access_config = nil)
```

Comment:
Providing some context. The following is huge IMO, so please, correct me if I'm wrong here.

I implemented this method because I wanted `server.insert` to properly work with network models #201, my reasoning for the method was:
- Interface config and network resource do not match each-other well
- `server.insert` was getting overloaded, there were 30 additional lines to the method that very awkwardly handled the network.
- what is better aware of what network looks like than the model itself? 

Hence the placement in network model.
I want you to consider this before returning back to generating arrays in the insert code.